Packs/DigitalShadows/Layouts/layoutscontainer-Digital_Shadows_Intelligence_Incident_Layout.json

```diff
@@ -0,0 +1,334 @@
+{
```

Comment:
@ds-jdcockrill Why do we need another layout? They both look the same.

Reply:
They are slightly different - the Client incident doesn't have 'Impact' or 'Mitigation' data and so the central panel in the layout omits those fields.

Reply:
I see. It seems like a tiny difference between the 2. Also, in case they are empty , they would not show up in the layout (unless the user click on "show empty fields") - seems unneeded to me.
BTW, you wrote that the "Client" incident doesn't have these fields but I can see that it is the Client layout and not the intelligence layout that contain these fields, so please make sure you didn't confuse between them.

Reply:
Hi Tomer,

You are correct, Client Incident is the one that has 'Impact' and 'Mitigation' on it - this is the correct way round and it's just my comment that is incorrect. Having thought about it, I'd like to keep the layouts separate as it is probable that in a subsequent update we will do a bit more work on the layouts and playbooks associated with each. The intelligence incident will more likely end up looking like a threat intelligence report that can be processed in slow-time whereas the client incident is more of a real-time threat that needs investigating and addressing.